src/renderers/dom/shared/__tests__/ReactMount-test.js

```diff
@@ -297,18 +295,6 @@ describe('ReactMount', () => {
     expect(calls).toBe(5);
   });
 
-  it('tracks root instances', () => {
-    // Used by devtools.
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(0);
-    ReactTestUtils.renderIntoDocument(<span />);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-    var container = document.createElement('div');
-    ReactDOM.render(<span />, container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(2);
-    ReactDOM.unmountComponentAtNode(container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-  });
-
```

Comment:
I'm less worried about those since the source is obviously devtools-related, unlike the name _instancesByReactRootID which I deleted at least once accidentally when I thought it was unused.